build-release.sh

```diff
@@ -68,16 +68,16 @@ echo "--- Building Release: ${VERSION}"
 
 mkdir -p "$OUT_DIR"
 
-build mac     ""   GOOS=darwin  GOARCH=amd64
-build win32   .exe GOOS=windows GOARCH=386
-build win64   .exe GOOS=windows GOARCH=amd64
-build linux32 ""   GOOS=linux   GOARCH=386    CGO_ENABLED=0
-build linux64 ""   GOOS=linux   GOARCH=amd64  CGO_ENABLED=0
-
-package linux32 tgz
-package linux64 tgz
-package linux64 rpm
-package linux64 deb
-package mac     zip
-package win32   zip
-package win64   zip
+build osx-amd64     ""   GOOS=darwin  GOARCH=amd64
+build osx-arm64     ""   GOOS=darwin  GOARCH=arm64
```

Comment:
There's no such thing as "OS X" anymore, so this should probably be:

```suggestion
build macos-amd64     ""   GOOS=darwin  GOARCH=amd64
build macos-arm64     ""   GOOS=darwin  GOARCH=arm64
```

And same for `package` below.